test/dlc_tests/container_tests/bin/pytorch_tests/testTorchdata

```diff
@@ -19,7 +19,7 @@ git clone $TORCHDATA_REPO \
 sed -i "s/awscli.*//g" data/test/requirements.txt
 
 echo "Installing pre-requisites."
-pip install -r data/test/requirements.txt
+pip install -r data/test/requirements.txt awscli==1.29.8 pyyaml==6
```

Comment:
Please change the awscli installation constraint to be `>=x` rather than `==x`. Same for pyyaml if possible.

Reply:
ack